src/styles/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: $calendar-size;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all;
+    transition-duration: 0.5s;
+
+    box-sizing: border-box;
+    height: $column-size;
+    width: $column-size;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
+    border: 1px solid black;
+
+    &:hover {
+      transition-property: transform, background-color;
```

Comment:
The 'transition-property' should be defined in the original state of the element, not in the hover state. This ensures that the transition is properly defined for both the hover-in and hover-out events.